.tekton/pipeline/currency-bot-pipeline.yaml

```diff
@@ -106,6 +106,10 @@ spec:
           value: $(params.npm-version)          
         - name: skip-cache
           value: "false"
+        - name: prerelease
+          value: "false"
+        - name: prerelease-node-version
+          value: ""
```

Comment:
This can be removed. The defaults should kick in.

Reply:
If we remove this, we will get tekton lint error

Reply:
resolved